components/server/src/prebuilds/github-app.ts

```diff
@@ -112,6 +114,19 @@ export class GithubApp {
                     res.redirect(301, this.getBadgeImageURL());
                 });
 
+        // Use RequestContext
+        options.getRouter &&
+            options.getRouter().use((req, res, next) => {
+                runWithRequestContext(
+                    {
+                        requestKind: "probot",
+                        requestMethod: req.path,
+                        signal: new AbortController().signal,
```

Comment:
in theory one can bind `req.on('aborted', () => abortController.abort())` but probably here it is not desirable, we still want to process even if client cancelled?

Reply:
Yeah. I removed that line, just to avoid any unwanted side-effects of this PR :see_no_evil: